greenheart/tools/eco/electrolysis.py

```diff
@@ -33,37 +32,54 @@ def run_electrolyzer_physics(
     electrolyzer_size_mw = greenheart_config["electrolyzer"]["rating"]
     electrolyzer_capex_kw = greenheart_config["electrolyzer"]["electrolyzer_capex"]
     
+    # IF GRID CONNECTED
     if greenheart_config["project_parameters"]["grid_connection"]:
-        energy_to_electrolyzer_kw = np.ones(365 * 24 - 4*7*12) * ( # TODO why the subtraction here?
-            electrolyzer_size_mw * 1e3
-        )
+        n_pem_clusters = 1
```

Comment:
well either we need n_pem_clusters or cluster_size_mw - either one would be okay.

Reply:
at this point - it has to be hard-coded for grid-connected cases when the hydrogen demand is input (rather than an input power signal). Hydrogen demand is input for grid-connected cases. In the future, it'd be nice to integrate cluster-level control strategies for this case - but that'll be in BERT. I agree - but the electrolyzer code won't work as expected for grid-connected cases if n_pem_clusters isn't set to 1.